classes/Search.php

```diff
@@ -137,7 +137,23 @@ class SearchCore
 
     public static function extractKeyWords($string, $id_lang, $indexation = false, $iso_code = false)
     {
-        $sanitizedString = Search::sanitize($string, $id_lang, $indexation, $iso_code, false);
+/*         start improvement
```

Comment:
Very bad comment style :(

Reply:
Sorry, will pay more attention next time :-(

Reply:
No need to be sorry, the great thing is maybe today was an opportunity to grow to your skills :D